src/manifest.js

```diff
@@ -100,17 +100,27 @@ export const setupMediaPlaylist = ({ playlist, uri, id }) => {
  * @param {Object} master
  *        The master playlist
  */
-export const setupMediaPlaylists = (master) => {
+export const setupMediaPlaylists = (master, type) => {
```

Comment:
I think this function shouldn't consider the type itself, as that logic may belong more in the appropriate loader. Instead we can have something like `useNameForId` or something like that. JSDoc can also use an update.